.github/workflows/test.yaml

```diff
@@ -53,7 +83,14 @@ jobs:
         run: npm install -g @fairdatasociety/fdp-play
 
       - name: Run fdp-play
-        run: fdp-play start -d
+        run: fdp-play start -d --fresh --blockchain-image fairdatasociety/fdp-contracts-blockchain:1.1.1
```

Comment:
you don't need this change as fdp-play always will have the freshest fdp-contracts-blockchain image by default

Reply:
I wasn't sure if the tests didn't work because of the latest image, so I tried using a specific version. But that didn't help either.